api/StyleSheetList.json

```diff
@@ -20,7 +20,7 @@
             "version_added": "31"
           },
           "ie": {
-            "version_added": null
+            "version_added": "8"
```

Comment:
I tested https://mdn-bcd-collector.appspot.com/tests/api/Document/styleSheets in IE8, but no earlier version. Now I tested it and it return true back to IE6. Can you check if `document.styleSheets` is a `StyleSheetList` instance in IE6 and if so update this?